docs/dynamicproxy.md

```diff
@@ -18,6 +18,7 @@ If you're new to DynamicProxy you can read a [quick introduction](dynamicproxy-i
 * [Use proxy generation hooks and interceptor selectors for fine grained control](dynamicproxy-fine-grained-control.md)
 * [SRP applies to interceptors](dynamicproxy-srp-applies-to-interceptors.md)
 * [Behavior of by-reference parameters during interception](dynamicproxy-by-ref-parameters.md)
+* [Some default parameter values may not be reproduced correctly in proxy types](dynamicproxy-default-parameter-values.md)
```

Comment:
It might be better if the docs page title was something shorter like "Optional parameter value limitations".